OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -178,6 +195,21 @@ open class OptimoveConfigBuilder: NSObject {
             Logger.error(error.localizedDescription)
         }
 
+        return self
+    } 
+
+    @discardableResult func setCredentials(
+        optimoveCredentials: String?,
+        optimobileCredentials: String?,
+        preferenceCenterCredentials: String?
+    ) -> OptimoveConfigBuilder {
+        setCredentials(optimoveCredentials: optimoveCredentials, optimobileCredentials: optimobileCredentials)
+        
+        if let preferenceCenterCredentials = preferenceCenterCredentials, !preferenceCenterCredentials.isEmpty {
+            self.preferenceCenterCredentials = preferenceCenterCredentials
+            features.insert(.preferenceCenter)
```

Comment:
the point of features was separating which credentials they have now VS which features they want in general. so, when they init initially, they say we want features A, B, C. Then they can provide credentials for them later

hence, 
- no need to add a feature when setting credentials
- if set credentials for feature that was not requested -- error